src/requests/debug.cpp

```diff
@@ -19,51 +19,50 @@
 
 namespace graft { namespace requests { namespace debug {
 
+GRAFT_DEFINE_IO_STRUCT(DbSupernode,
+    (std::string, Address),
+    (uint64, StakeAmount),
+    (uint64, LastUpdateAge)
+);
+
+GRAFT_DEFINE_IO_STRUCT(SupernodeListResponse,
+    (std::vector<DbSupernode>, items)
+);
+
+
+
+
+GRAFT_DEFINE_JSON_RPC_RESPONSE_RESULT(SupernodeListJsonRpcResult, SupernodeListResponse);
```

Comment:
Why this is needed?  why do we care to send json-rpc in this particular case when there is no client that wants it?